### DISCUSSION TITLE
Welcome to addon-cloudflared Discussions!

### DISCUSSION THREAD

**Discussion by brenner-tobias** [homeassistant-apps/app-cloudflared#187] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other users and developers of the Cloudflared Home Assistant Add-On. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas
  * Make feature requests
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by theloquitur** [comment]
Hi,

When I went and added the CloudFlare repository, I made the mistake of not having removed DuckDNS and Let's Encrypt.  Now, when I try to use the CloudFlare addon, I get an "400: Bad Request" error.  I'm writing to ask what I need to do on the CloudFlare side to start over again, as this seems to be the simplest route, that and without DuckDNS and Let's Encrypt.

Thanks,
Rich
